x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S=${WORKDIR}/rofi-${PV}+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun test +windowmode"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/glib:2
```

meson.build says: 
```
glib_min_major=2
glib_min_minor=40
glib_min_version='@0@.@1@'.format(glib_min_major, glib_min_minor)
plugins_deps = [
    dependency('glib-2.0', version: '>= @0@'.format(glib_min_version)),
```
```suggestion
	dev-libs/glib-2.40:2
```

And yes, I know you just copied this from the rofi ebuild from ::gentoo. But thay aren't always perfect.